site/_posts/2022-07-15-remix-demo.md

```diff
@@ -85,13 +86,13 @@ Here's an example of the `.env` file, with values filled out. Some of your value
 ```
 CLIENT_ID="85a03867-dccf-4882-adde-1a79aeec50df"
 CLIENT_SECRET="b4xOdsBUWHQkkU3BOqAxSilfttI4TJv9eI_LOj8zVgE"
-AUTH_URL="http://localhost:9000"
+AUTH_URL="http://localhost:9011/oauth2"
 AUTH_CALLBACK_URL="http://localhost:3000/auth/callback"
 ```
 
 ## Test it out
 
-At this point you should be ready to fire up the Remix server. Do so by running `npm start` in the directory where your `.env` file is.
+At this point you should be ready to fire up the Remix server. Do so by running `npm run dev` in the directory where your `.env` file is.
```

Comment:
looks like `npm run dev` is dev mode and `npm start` is prod mode. Any reason to pick one over the other?

Reply:
`npm start` does not work at this stage, because the Remix app is not built yet. We either have to build it first now, or start it with `npm run dev`

Reply:
Sounds good.